src/components/ReservedUserData/ReservedUserData.tsx

```diff
@@ -0,0 +1,186 @@
+import { ReservedUserDataFragment } from "./ReservedUserDataFragment/ReservedUserDataFragment";
+import { IoIosArrowDown } from "react-icons/io";
+import React, {useContext, useEffect, useState} from "react";
+import {API_URL} from "../../config/apiUrl";
+import {FilterContext} from "../../contexts/filter.context";
+import {ContractType, Internship, TypeWork} from "../../../../HeadHunter_server/types/student/student.enum";
```

Comment:
Typy z be. To nie przejdzie na serwerze ...

Reply:
zastanawiam sie czy nie lepiej to poprawić dopiero na main, a na developie zostawić tak jak to powinno być